inst/include/types.hpp

```diff
@@ -123,15 +160,32 @@ struct IntermediateData {
   Tensor1<real_type> Xhivn_incagerr;
   Tensor2<real_type> hiv_deaths_age_sex;
   Tensor3<real_type> grad;
+  Tensor4<real_type> gradART;
+  Tensor2<real_type> artelig_hahm;
+  Tensor1<real_type> hivpop_ha;
   real_type cd4mx_scale;
   real_type artpop_hahm;
   real_type deaths;
   int everARTelig_idx;
   int cd4elig_idx;
   real_type infections_a;
   real_type infections_ha;
+  real_type deaths_art;
+  real_type Xart_15plus;
+  real_type Xartelig_15plus;
+  real_type expect_mort_artelig15plus;
```

Comment:
these names are grim

Reply:
it's possible that we might refactor things later into rather than a single struct where the naming convention does all the work into a set of nested structs?

Reply:
Yeah I think that is a good idea